vars/Debian.yml

```diff
@@ -1,3 +1 @@
 sshd_service_name: ssh
-ssh_owner: root
```

Comment:
I'm not really happy with this change.
Before this, there was only one place for every OS, where the owner and group were defined - the vars-file.
Now for some OS the owner is in the defaults and group is in vars.

I'd rather have these on the same place and since this cannot be the defaults, it should be vars.

Reply:
Fixed in 7f1331.